osquery/config/config_tests.cpp

```diff
@@ -16,23 +17,52 @@
 #include <osquery/registry.h>
 #include <osquery/sql.h>
 
+#include <boost/filesystem/operations.hpp>
+
 #include "osquery/core/test_util.h"
 
 namespace osquery {
 
 // The config_path flag is defined in the filesystem config plugin.
 DECLARE_string(config_path);
+const std::string kFakeDirectory = "/tmp/osquery-fstests-pattern";
 
 class ConfigTests : public testing::Test {
  public:
   ConfigTests() {
     FLAGS_config_plugin = "filesystem";
     FLAGS_config_path = kTestDataPath + "test.config";
+  }
+
+ protected:
+  void createFileAt(const std::string loc, const std::string content) {
+    std::ofstream test_file(loc);
+    test_file.write(content.c_str(), sizeof("test123"));
+    test_file.close();
+  }
+
+  void SetUp() {
```

Comment:
Since you're using twice, instead of repeating code (even though it's test code), what about putting it into `core/test_util.cpp`?